spec/features/dashboard_feature_spec.rb

```diff
@@ -27,6 +27,7 @@
       visit authenticated_root_path
 
       el = find("#download-kubeconfig")
+      wait_until { !find("#download-kubeconfig")[:disabled] }
```

Comment:
This line and the following can just be written as `expect(el).not_to match_css('[disabled]')` or `expect(el).to match_css('not([disabled])')`